src/api/index.js

```diff
@@ -331,3 +331,14 @@ export function getInstallProperties() {
   const uri = `${apiRoot}/v1/properties`;
   return get(uri);
 }
+
+export function shouldDisplayLogout() {
+  console.log('toad skin');
```

Comment:
🐸

Reply:
🤣 oops, now the world knows my most common debug statement